.vscode/launch.json

```diff
@@ -264,8 +318,16 @@
       ],
       "stopOnEntry": false,
       "sourceMaps": true,
+      "smartStep": true,
       "outFiles": ["${workspaceFolder}/packages/*/out/**/*.js"],
-      "preLaunchTask": "Compile"
+      "sourceMapPathOverrides": {
+        "../../../test/vscode-integration/": "${workspaceFolder}/packages/salesforcedx-vscode-apex-debugger/out/test/vscode-integration/*",
+        "../../../../test/vscode-integration/": "${workspaceFolder}/packages/salesforcedx-vscode-apex-debugger/out/test/vscode-integration/*",
+        "webpack:///salesforcedx-utils-vscode/./*": "${workspaceFolder}/packages/salesforcedx-utils-vscode/*",
+        "webpack:///*": "*"
+      },
+      "preLaunchTask": "Compile",
+      "internalConsoleOptions": "openOnSessionStart"
     },
     {
       "name": "Launch SObject Generator Tests",
```

Comment:
The sobjects-faux-generator package has 3 different configs; 1 that runs both integration & unit tests, 1 for integration tests and another for unit tests. Do we want to consolidate the configs into only one that runs both integration & unit tests or have a dedicated launch config for every test category ?

Reply:
I think it would be helpful to keep the test configs separate for each category in case we don't need to run both integration and unit tests at the same time.

Reply:
Cool, I'll remove the option that runs both test categories at once so we have the same configs across modules.

Reply:
I agree. We should have one for unit and one for integration and that's it.